/clip_scene_rec:
  ros__parameters:
    callback_period_sec: 1.0
    scene_labels: ['indoor', 'outdoor', 'transportation']
    scene_descriptions: ['inside a building', 'outdoors', 'inside a vehicle']
    clip_model: 'ViT-B/32'